manifests/istio-control/istio-discovery/templates/mutatingwebhook.yaml

```diff
@@ -3,11 +3,7 @@
 apiVersion: admissionregistration.k8s.io/v1beta1
 kind: MutatingWebhookConfiguration
 metadata:
-{{- if eq .Release.Namespace "istio-system"}}
   name: istio-sidecar-injector{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}
-{{ else }}
```

Comment:
note that this change will break things now that https://github.com/istio/istio/pull/22828 has been merged